src/components/home/InputForm/InputForm.tsx

```diff
@@ -51,9 +66,15 @@ export function InputForm() {
   }, [localStorage, setBodyInfo]);
 
   useEffect(() => {
-    if (gender === Gender.Male) setValue('pregnancy', Pregnancy.None);
+    if (Number(gender) === Gender.Male) setValue('pregnancy', Pregnancy.None);
```

Comment:
don't need this.

Reply:
Because the model has been updated to enumerate without string, 
> Gender.Male will be an Integer (0, 1)

But the value from watch will not be converted according to the model,
so it will be the string format of the original select option value ("0", "1")